mediabridge/integrations/ollama_api.py

```diff
@@ -1,7 +1,16 @@
+import os
+
 import requests
+from dotenv import load_dotenv
 from requests.auth import HTTPBasicAuth
 
-AUTH = HTTPBasicAuth("ollamauser", "INSERT_PASSWORD_HERE")
+load_dotenv()
+
+OLLAMA_PASSWORD = os.getenv("OLLAMA_PASSWORD")
+if not OLLAMA_PASSWORD:
+    raise ValueError("OLLAMA_PASSWORD is not set in the environment or .env file.")
```

Comment:
This is the right idea!

But I think it would be better to do this inside the function that uses the password. The reason being, the codebase is currently highly modular and decoupled. What I mean by that is that someone can run "main" method or another, and do a specific set of tasks, without touching the other 75% of the code. So we might want to allow developers to run, say, ratings loading and the other recommend function withou running this. The way you have it now, as soon as this file is imported it will check for the `OLLAMA_PASSWORD` and crash the program if it doesn't find it.

Additionally, it's good practice to hide the username as well, for added "security". That way, someone can't say "Oh I have the username, all I need to do is guess the password".